Add "Grenfell Tower Fire" promo section

pmanrubia opened the pull request:
Replaces `GOV.UK blogs` with `Grenfell Tower Support` in Home page promo Section

### Before
![image](https://user-images.githubusercontent.com/227328/27436553-88298e32-5757-11e7-82cc-57a2a41d0800.png)

### After

![image](https://user-images.githubusercontent.com/227328/27441316-0facba70-5765-11e7-836d-475075b7b2a4.png)

nickcolley commented:
![screen shot 2017-06-22 at 15 31 19](https://user-images.githubusercontent.com/2445413/27439550-10bceade-5760-11e7-84a4-a26f6495e565.png)

Updated image ^

pmanrubia commented:
I have removed the words `information` and `and` from the link title as requested by the Content Team. I have rebased with `master` and force pushed in preparation for merging.
